saleor/tax/calculations/order.py

```diff
@@ -135,10 +138,11 @@ def update_taxes_for_order_lines(
                 default_tax_rate,
                 country_code,
             )
-        elif line.tax_rate is not None:
-            # line.tax_class can be None when the tax class was removed from DB. In
-            # this case try to use line.tax_rate which stores the denormalized tax rate
-            # value that was originally used.
+        elif line.tax_class_name is not None and line.tax_rate is not None:
```

Comment:
Do we need to have here `line.tax_class_name is not None and line.tax_rate is not None`?
Can't we replace it with `line.tax_rate is not None`. This is the value that you're using in 146. So no need to check `line.tax_class_name`, right?

Reply:
Yes, tax_class_name determines that the tax class was ever set here before; in case of draft orders if it was removed from the system (edge-case) we still want to use saved `line.tax.rate`.